### ISSUE TITLE
'Info' icons containing tooltip information is not accessible with the keyboard present under 'Settings' blade.

### ISSUE THREAD

**Issue by ato9000** [Azure/cosmos-explorer#1036]
https://msdata.visualstudio.com/CosmosDB/_workitems/edit/1239259

**Comment by ato9000** [comment]
Note: User credentials should NOT be included in the bug. 

Pre-Requisite:
Select "Core(SQL)" in API combo box while creating the Cosmos DB.

Repro Steps:
Open URL: https://ms.portal.azure.com/ and login using valid credentials.
Navigate to "Azure Cosmos DB " from left pane and select it.
Navigate to any of the Azure Cosmos DB which is already created and select it.
Navigate to "Data Explorer" from the left and select it.
Navigate  to 'Settings' menu item and select it.
Navigate through all the controls under Settings blade.
Press tab key to navigate over radio buttons and edit fields having info icon containing tooltip information.
Press shortcut key F1 or Shift+F1 to access the info icon tooltip information.
Verify the issue.

Actual Result:
'Info' icons containing tooltip information is not accessible with the keyboard present under 'Settings' blade.

Expected Result:
'Info' icons containing tooltip information which are accessible using mouse hovering  should also be accessible through keyboard using F1 key present under 'Settings' blade.


Note:
Similar issue exits for all info icon controls under settings blade.
This issue exists throughout the application.

https://user-images.githubusercontent.com/8519407/132719071-744891c7-ad6a-4419-b8d1-87ce108a51bb.mp4
